korge/src/commonMain/kotlin/com/soywiz/korge/tests/ViewsForTesting.kt

```diff
@@ -222,8 +223,9 @@ open class ViewsForTesting(
 	}
 
     @Suppress("UNCHECKED_CAST")
-    inline fun <reified S : Scene> sceneTest(
+    inline fun <reified S : Scene> ViewsForTesting.sceneTest(
```

Comment:
Why is this receiver required?

Reply:
Copy and paste mistake from my extension function, sorry.